make/InitSupport.gmk

```diff
@@ -313,6 +313,11 @@ else # $(HAS_SPEC)=true
       SOURCE_DATE := $$(shell $$(DATE) +"%s")
     endif
     export SOURCE_DATE_EPOCH := $$(SOURCE_DATE)
+    ifeq ($$(IS_GNU_DATE), yes)
+      SOURCE_DATE_ISO_8601 := $(shell $(DATE) --utc --date="@$(SOURCE_DATE_EPOCH)" +"%Y-%m-%dT%H:%M:%SZ" 2> /dev/null)
```

Comment:
Are you getting this to work without `export`? I would have expected that to be needed.

Reply:
just checking log now

Reply:
you're right !

Reply:
@erikj79 right, so I was using the wrong configuration and reproducible-build was not set, now I am, I can see I am where I was before in that SOURCE_DATE_ISO_8601 being set in InitSupport.gmk is not visible in JarArchive.gmk, i've tried both with and without the export...?

Reply:
@erikj79 fixed it I think, problems is above not enough $'s !! for $(shell.......

Reply:
resolved, thanks

Reply:
Ah, the default diff view didn't show me enough context. This is part of a macro body, then yes, you need to double all the $ to get it evaluated at the correct time.